src/lib_python/ViZDoomMethodsDocstrings.h

```diff
@@ -893,6 +893,33 @@ See also:
 
 Note: added in 1.1.9.)DOCSTRING";
 
+    const char *setCategoryMapping = R"DOCSTRING(Sets a custom category mapping for object classification.
```

Comment:
I think if the class is obtained from WAD data, then changing it should be done when editing the WAD file, and this layer of API is not needed.